README.md

```diff
@@ -0,0 +1,11 @@
+# Learn-OCaml autogen
+
+This is Learn-OCaml autogen, a utility designed to provide a simple way to
```

Comment:
Please update the README.md file to be more specific about what autogen brings. By reading the first sentence, a user should be able to understand if she needs this tool, or not.